src/System.Diagnostics.StackTrace/src/System/Diagnostics/StackTraceSymbols.CoreCLR.cs

```diff
@@ -0,0 +1,244 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+extern alias SRE; 
```

Comment:
This was needed because S.R.E is a partial facade and it has both Path and BitConverter as full types instead of forwards to mscorlib.  Without this we get errors because a type is defined in two places.  @ericstj was a little concerned here because it means that any S.R.E type we use which is defined in both places we will now silently prefer the mscorlib version, but I don't know of a better way around this  (/cc @weshaggard).